src/grandorgue/GOApp.cpp

```diff
@@ -140,7 +169,7 @@ bool GOApp::OnInit() {
   SetTopWindow(m_Frame);
   m_Log = new GOLog(m_Frame);
   wxLog::SetActiveTarget(m_Log);
-  m_Frame->Init(m_FileName);
+  m_Frame->Init(m_FileName, m_IsGuiOnly);
```

Comment:
@oleg68 I don't like modes that are too strict. Do you think it should be a good idea to rename this m_LoadingMode? 

This should supports some headless mode, gui only mode, ... 

In a more flexible way

Reply:
Usually I defer adding complexity until it becomes necessary. So I'd leave the boolean `IsGuiOnly` now.

When we introduce another loading features we will rename it to m_LoadingFlags and we will change it's type to `unsigned`. But now I consider making a one-bit mask as unnecessary complexity.

Reply:
Personally I think this "no samples" just gui command line option is close to meaningless (especially considering the GUI displaying/editing capabilities of GOODF). The opposite, a headless mode, would in my opinion be much more usable for hardcore command line users who wish to build a GO driven organ console without any un-necessary resource consumption or screen/monitor.

Many GO users, even including some of the sample set developers, have a really hard time using the command line at all, believe it or not! More "normal" users would likely discover/use this option of loading just the gui if there would be a checkbox in the File->Settings instead that would affect the same thing when checked. Also much nicer to not have to re-start GO to enter/exit the "just gui" mode.

Reply:
@oleg68 @rousseldenis Sorry, scrap the last part of my comment above. It's already possible to achieve this "just-gui" from within the GO gui in the File->Settings dialog by setting the sample loading option to "Don't load" (instead of Stereo or Mono).

Still I cannot but question the value of this command line feature...

Reply:
@larspalo Using `Don't load` is not comfortable because this option is remembered so It needs to be reverted every time when a user wants to load an (another) organ with sound samples.

So I consider the `-g` command-line option as a lightweight solution for temporal disabling loading samples. 

I agree that capability of altering any program setting (not just this) from the command line would be a better solution, but it would require a lot of coding.

> Still I cannot but question the value of this command line feature

I think It is not big enouth for a lot of coding. But this implementation is very simple, so the value/cost relation allows to introduce it.

Reply:
> so It needs to be reverted every time when a user wants to load an (another) organ with sound samples.

And using commandline -g, GO needs to be quitted every time the user wants to load another organ with samples! It's exactly the same trouble - only worse and even less comfortable for any normal user!

But since you've already implemented it, and I don't think it will cause any harm, I'll agree to this rather pointless PR.